srcpkgs/base-system/template

```diff
@@ -2,7 +2,7 @@
 pkgname=base-system
 reverts="0.113_1"
 version=0.112
-revision=6
+revision=7
```

Comment:
please bump version instead, and reset rev

Reply:
done, please review/merge